examples/howto/plot_record_extracellular_potentials.py

```diff
@@ -98,9 +98,9 @@
 # Add spike raster to subplot
 net.cell_response.plot_spikes_raster(ax=axs[2], show=False)
 
-# Finally, add the CSD to the bottom subplot
-net.rec_arrays['shank1'][trial_idx].smooth(window_len=window_len).plot_csd(
-    ax=axs[3], show=False)
+# Finally, add the CSD to the bottom subplot, with the colormap ranging from -0.02 to 0.02 µV,
```

Comment:
Was this left over from when these options were exposed in the plot call?

Or did you intentionally add it _because_ we decided to remove these options for simplicity?

Your call but I just wanted to point it out

Reply:
Whoops no, that was supposed to go. Changed it!